src/main/java/frc/robot/subsystems/PowerPanel.java

```diff
@@ -31,7 +30,7 @@ public PowerPanel() {
 	@Override
 	public void periodic() {
 		// put the current draw on the SmartDashboard
-		SmartDashboard.putNumber("PDH Current (Amps)", PDH.getTotalCurrent());
+		//SmartDashboard.putNumber("PDH Current (Amps)", PDH.getTotalCurrent());
 
 		// TODO: put the Energy draw on the SmartDashboard
```

Comment:
Don't you guys want to know what the current draw is?

Reply:
I think we were trying to reduce things sent to SmartDashboard, and this wasn't absolutely necessary.